modules/planning/tasks/deciders/open_space_fallback_decider.cc

```diff
@@ -48,35 +48,25 @@ Status OpenSpaceFallbackDecider::Process(Frame* frame) {
     *(frame_->mutable_open_space_info()->mutable_fallback_trajectory()) =
         frame->open_space_info().chosen_paritioned_trajectory();
     auto fallback_trajectory_vec =
```

Comment:
use another variable name. It's a traj and gear pair actually

Reply:
> use another variable name. It's a traj and gear pair actually

fixed